### ISSUE TITLE
Review Ticket - Remember Forever Nighthawk Quiz

### ISSUE THREAD

**Issue by JasonGuan1012** [JasonGuan1012/JasonCSA_2025#1]
# This is before remember forever command
![image](https://github.com/user-attachments/assets/85250a2d-2748-435d-b2f9-b10f62888965)

# This is after remember forever command
![image](https://github.com/user-attachments/assets/6668ed3c-3ce7-4417-812e-317cf446b9e0)

# This is showing a localhost running instance in browser and putting the name on the home page.
![image](https://github.com/user-attachments/assets/4e85204e-d55d-4212-83e8-a058c00521fb)

# Team Pic Proof
![image](https://github.com/user-attachments/assets/220b5879-3e36-4028-947f-e016316c176a)

# What does rm do?
- In simple terms, rm is a command that is used to remove files and directories. In this case, we removed the 'nighthawk' directory.
